lambda/lambda_function.py

```diff
@@ -136,7 +100,7 @@ def ask(self, question: str, context: dict = {}) -> dict:
 class BaseRequestHandler(AbstractRequestHandler):
     """Base class for request handlers."""
 
-    question = LLMQuestionProxy()
+    question = LLMQuestionProxy(Client(LLM_URL, LLM_KEY, LLM_MODEL))
```

Comment:
```suggestion
    question = LLMQuestionProxy(LlmApiClient(LLM_URL, LLM_KEY, LLM_MODEL))
```

To respect the changing of the class name.